### DISCUSSION TITLE
Regarding Add Layer

### DISCUSSION THREAD

**Discussion by wilfredkisku** [fastmachinelearning/hls4ml#731] [General]
While ```Concatenate``` and ```Merge``` have been supported, is there the support for ```Add ``` Layer yet?

**Comment by vloncar** [comment]
Did you try it?

**Reply by wilfredkisku** [reply]
I was made aware that ```Merge``` and ```Concatenate``` layers which weren't earlier available have been made recently. Its just a question about the Layer whether it has been added or not since I couldn't locate it in the source code.

**Reply by vloncar** [reply]
Try it.
